io/common/bootlist_test.py.data/bootlist_test_network.yaml

```diff
@@ -0,0 +1 @@
+host_interfaces: "None"
```

Comment:
Is this right ?

Reply:
this means test will still work if host_interfaces and disks has default None  . if not may be we should have meaningful names instead of None

Reply:
yes test is working,i didn't check that time.i need make this as empty string.